src/classes/CW_BatchUpdateAudienceGroupMember.cls

```diff
@@ -0,0 +1,69 @@
+global class CW_BatchUpdateAudienceGroupMember implements Database.Batchable<sObject>{
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.